docs/features/alerting.md

```diff
@@ -52,6 +61,54 @@ alerting:
         receiver_url: https://bots.keybase.io/webhookbot/<Your-Keybase-Hook-Token>
 ```
 
+## Example With Authentication
+
+For example, if you would like to receive notifications in your custom webhook authenticated with a bearer token taken from an environmental variable whenever Connaisseur admits a request to your cluster, your alerting configuration would look similar to the following snippet:
+
+```
+alerting:
+  admit_request:
+    templates:
+      - template: ecs-1-12-0 
+        receiver_url: https://your.custom.domain.com/webhook/admit
+        receiver_authentication_type: bearer
+        receiver_authentication_bearer: 
+          token_env: CONNAISSEUR_ADMIT_REQUEST_WEBHOOK_AUTH_TOKEN
+```
+
+You then have to set the `CONNAISSEUR_ADMIT_REQUEST_WEBHOOK_AUTH_TOKEN` environment variable referencing the bearer token secret you want to use into the connaisseur deployment.
+
+Or if you would like to use the service account token as the bearer token, you can use the following snippet:
+
+```
+alerting:
+  admit_request:
+    templates:
+      - template: ecs-1-12-0 
+        receiver_url: https://your.custom.domain.com/webhook/admit
+        receiver_authentication_type: bearer
+        receiver_authentication_bearer: 
+          token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
This is just an example of loading the token from the service account. Since the service account token is just a file inside the pod I prefer not to explicitly have a bool to control this behavior but to let it generic.

For what concern the service account as authentication method, I am not sure if it makes sense to use it in a production environment with the webhook inside the cluster (To not let the token to leak outside). 
I have just tried it to experiment with the `TokenReview` and the `system:auth-delegator` cluster role.

Taking the token from a file may be used in general if the secret has been mounted as file inside the pod.

To make the use case more clear I can change the example to something more generic like `/etc/webhook/token`.